src/data/config/train_config.cfg

```diff
@@ -0,0 +1,44 @@
+# Training cfg
```

Comment:
very relevant file. contains the config for the "train_flow" (= ds down; train; upload unconfident subset)